### DISCUSSION TITLE
New firmware

### DISCUSSION THREAD

**Discussion by makkie2002** [goecharger/go-eCharger-API-v2#186] [General]
My Go-e charger updated today to 055.8. Does anybody know were I can find the changelog?

**Comment by 0xFEEDC0DE64** [comment]
Isnt it visible in the app? Usually we put it there

**Comment by mleck28** [comment]
Hello, I am currently on firmware 56.1 and unfortunately, I also see no information in the change log about what has been changed.

![IMG_20240405_190033](https://github.com/goecharger/go-eCharger-API-v2/assets/160210695/9cf855e0-c137-4843-9515-1c4e1a1605d0)

**Reply by IamMattM** [reply]
Same for me. 
No CHANGELOG on 56.2:
![image](https://github.com/goecharger/go-eCharger-API-v2/assets/29542928/b615f9b9-4217-4141-82dd-ee597043f900)

**Comment by 0xFEEDC0DE64** [comment]
Hi is this still relevant? I think we have updated the changelog at some point.

**Reply by IamMattM** [reply]
It seems to be fixed  and the changelog is now showing in the app.
